src/components/kytos/misc/StatusMenu.vue

```diff
@@ -0,0 +1,764 @@
+    }
+    if (JSON.parse(localStorage.getItem('kytos/ui/selectedSwitches'))) {
+      this.selectedSwitches = JSON.parse(localStorage.getItem('kytos/ui/selectedSwitches'));
+    }
+    if (JSON.parse(localStorage.getItem('kytos/ui/selectedLinks'))) {
+      this.selectedLinks = JSON.parse(localStorage.getItem('kytos/ui/selectedLinks'));
+    }
+    if (JSON.parse(localStorage.getItem('kytos/ui/selectedInterfaces'))) {
+      this.selectedInterfaces = JSON.parse(localStorage.getItem('kytos/ui/selectedInterfaces'));
+    }
+  },
+  //Watches to see if data is changed to then store it within localStorage.
+  watch: {
+    switchTextFilter(newVal) {
+      localStorage.setItem('kytos/ui/switchTextFilter',  JSON.stringify(newVal))
+    },
+    linkTextFilter(newVal) {
+      localStorage.setItem('kytos/ui/linkTextFilter',  JSON.stringify(newVal))
+    },
+    interfaceTextFilter(newVal) {
+      localStorage.setItem('kytos/ui/interfaceTextFilter',  JSON.stringify(newVal))
+    },
+    currentSwitchData(newVal) {
+      localStorage.setItem('kytos/ui/currentSwitchData', JSON.stringify(newVal))
+    },
+    currentLinkData(newVal) {
+      localStorage.setItem('kytos/ui/currentLinkData', JSON.stringify(newVal))
+    },
+    currentInterfaceData(newVal) {
+      localStorage.setItem('kytos/ui/currentInterfaceData', JSON.stringify(newVal))
+    },
+    selectedSwitches: {
+      handler: function() {
+        localStorage.setItem('kytos/ui/selectedSwitches', JSON.stringify(this.selectedSwitches))
+      },
+      deep: true
+    },
+    selectedLinks: {
+      handler: function() {
+        localStorage.setItem('kytos/ui/selectedLinks', JSON.stringify(this.selectedLinks))
+      },
+      deep: true
+    },
+    selectedInterfaces: {
+      handler: function() {
+        localStorage.setItem('kytos/ui/selectedInterfaces', JSON.stringify(this.selectedInterfaces))
+      },
+      deep: true
+    },
+  },
+  filters: {
+    splitStatusReasons: function(statusReasons) {
+      return statusReasons.toString()
+    }
+  }
+}
+
+</script>
+
+<style lang="sass">
+
+@import '../../../assets/styles/variables'
+
+.k-status-menu
+  -webkit-order: 4
+  -ms-flex-order: 4
+  order: 4
+  display: flex
+  flex-direction: column
+  min-height: calc(100% - 45px)
+  padding: 10px
+  position: fixed
+  right: 0
+  top: 0
+  background-color: $fill-panel
+  width: calc(100% - 300px)
+  z-index: 998
+  box-shadow: 10px 0px 20px 5px rgba(0, 0, 0, 0.4)
+
+.k-status-title
+  display: flex
+  flex-direction: row
+  align-items: center
+
+  svg
+    fill: $fill-icon
+    width: 50px
+    height: 50px
+    padding: 10px
+    margin-right: 5px
+
+.panel-title
+  padding: 0
+  margin: 0
+  color: $fill-text
+
+  & > h1
+    font-size: 1.2em
+    font-weight: bold
+    color: $fill-text
+
+.k-status-wrapper
+  -webkit-flex: 1 1 auto
+  overflow-y: auto
+  height: 0px
+
+.data_status_table 
+  color: #ccc
+  max-height: 250px
```

Comment:
Since you removed the overflow-y and overflow-x, this max height was left behind, which was wrapping the table data to only 5 elements (that is why I was not seeing other switch's interfaces ;-) )

```suggestion
```